input/fsh/Condition.fsh

```diff
@@ -0,0 +1,90 @@
+// Condition Resources for Carl Frederickson
```

Comment:
it would be more readable if we separate these into different files and have one resource per file. this could also simply the logic in the bundle generator.